pkg/envoy/proxy.go

```diff
@@ -275,3 +282,14 @@ func (p *Proxy) buildCommand(ctx context.Context, cfgPath string) *exec.Cmd {
 
 	return cmd
 }
+
+// RemoveArgAndGetValue Function to get new args after removing given key
+// and also returns the value of key
+func RemoveArgAndGetValue(stringAr []string, key string) ([]string, string) {
+	for index, value := range stringAr {
+		if value == key {
```

Comment:
Do we need to worry about case sensitivity in this check?

Reply:
i guess not.